src/main/java/io/orangebeard/client/OrangebeardProperties.java

```diff
@@ -88,6 +90,17 @@ private void readPropertyFile(String name) {
         }
     }
 
+    private void readSystemProperties() {
+        this.endpoint = System.getProperty(ORANGEBEARD_ENDPOINT, this.endpoint);
+        try {
+            this.accessToken = System.getProperty(ORANGEBEARD_ACCESS_TOKEN) != null ? UUID.fromString(System.getProperty(ORANGEBEARD_ACCESS_TOKEN)) : this.accessToken;
```

Comment:
moet deze niet ook de default meekrijgen?

Reply:
als ik dat doe en hij is nog leeg, gaat die UUID.fromString een illegalargumentexceptie gooien lijkt me? Krijg je onnodige warnings als je met env vars werkt

Reply:
oh ja denk dat het zo idd wel goed gaat.